### ISSUE TITLE
Build/Installation error

### ISSUE THREAD

**Issue by jrgnicho** [ethz-asl/eclipsify#31]
I'm not sure that I'm installing this correctly but since this repo is setup as a ROS package (package.xml, and CMakeList.txt) I cloned this repository into a catkin-tools workspace and ran the build command as follows:
```
catkin build
```
and got the following error output
```
CMake Error at /my/local/path/eclipsify_ws/src/eclipsify/CMakeLists.txt:4 (find_package):
  By not providing "Findcatkin_simple.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "catkin_simple", but CMake did not find one.

  Could not find a package configuration file provided by "catkin_simple"
  with any of the following names:

    catkin_simpleConfig.cmake
    catkin_simple-config.cmake

  Add the installation prefix of "catkin_simple" to CMAKE_PREFIX_PATH or set
  "catkin_simple_DIR" to a directory containing one of the above files.  If
  "catkin_simple" provides a separate development package or SDK, be sure it
  has been installed.

```
I then ran rosdep to install the required binary dependencies however that didn't help
```
rosdep --from-path src --ignore-src  -r
```

I'm using ROS kinetic on ubuntu 16.04

**Comment by jrgnicho** [comment]
@HannesSommer thanks.   I'll try that out.

**Comment by HannesSommer** [comment]
This should be solved since the merge of #32 . Thanks again!
